.circleci/config.yml

```diff
@@ -47,6 +44,13 @@ workflows:
           filters:
             branches:
               only: master
+      - share-testing-image:
+          filters:
+            branches:
+              only:
+                - master
+          requires:
+            - e2e
```

Comment:
By the time this is merging to master, I'm assuming all the tests are good, and just doing one final e2e test to account for an unlikely race condition between merges on master of `ui` vs. `influxdb`.

Reply:
At what point does `e2e` go away? Once we cut over to triggering the workflow in `monitor-ci`?

Reply:
Yeah. We'll probably called it something more like `run-monitor-ci-tests` and it will just drop into where we are doing `e2e` now. Merges into `master` would still be required to complete `run-monitor-ci-tests` before they publish the `latest` image just to make sure everything is good.